### PR TITLE
PEP-257 bodhi.server.consumers.__init__.

### PR DESCRIPTION
**Pull request by bowlofeggs**
Signed-off-by: Randy Barlow <randy@electronsweatshop.com>

### CODE REVIEW

File: bodhi/tests/test_style.py [hunk 1]
```diff
@@ -47,9 +47,9 @@ def test_code_with_pydocstyle(self):
         The goal is that this test would one day check the entire codebase.
         """
         enforced_paths = [
-            'bodhi/server/scripts/__init__.py', 'bodhi/server/scripts/approve_testing.py',
-            'bodhi/server/scripts/clean_old_mashes.py', 'bodhi/server/scripts/expire_overrides.py',
-            'bodhi/server/scripts/initializedb.py']
+            'bodhi/server/consumers/__init__.py', 'bodhi/server/scripts/__init__.py',
```
Comment (ralphbean) [inline bodhi/tests/test_style.py]: Nonblocking comment here:

I recommend breaking these out into one item per line.

It makes it much easier to review -- to see what was added and what was changed.

**Review by ralphbean — APPROVED** [review]
LGTM.

**Review by ralphbean — CHANGES_REQUESTED** [review]
Eh, merge conflict now.  @bowlofeggs can you rebase?

**Review by ralphbean — APPROVED** [review]
Good to go.

File: bodhi/tests/test_style.py (hunk 1 above)
Comment (bowlofeggs) [reply bodhi/tests/test_style.py]: Fixed!

**Pull request merged**